conda_build_config.yaml

```diff
@@ -5,7 +5,7 @@ libprotobuf:
 protobuf:
   - 5.28.2
 spdlog:
-  - 1.14
+  - 1.15
```

Comment:
I am afraid we can't change a pinning in the `conda_build_config.yaml` without a full rebuild, as we could end with multiple packages that depend on the same `ros2-distro-mutex` that can't be installed.

Reply:
Let’s just use selectors for emscripten and not emscripten?

Reply:
Good idea!